x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/steps/blocks/action/processor_field_selector.tsx

```diff
@@ -73,7 +73,7 @@ export const ProcessorFieldSelector = ({
 
   const defaultLabel = i18n.translate(
     'xpack.streams.streamDetailView.managementTab.enrichment.processor.fieldSelectorSourceLabel',
-    { defaultMessage: 'Source field' }
+    { defaultMessage: 'Source Field' }
```

Comment:
Super pedantic nit, but I'd say the original of `Source field` is grammatically correct.

Reply:
yes, thank you fixed!